response_operations_ui/forms.py

```diff
@@ -57,6 +57,18 @@ def __init__(self, form, default_values=None):
             self.telephone.data = default_values.get('telephone')
 
 
+class EditCollectionExerciseDetailsForm(FlaskForm):
+    user_description = StringField('user_description')
+    period = StringField('period')
+    collection_exercise_id = HiddenField('collection_exercise_id')
+
+    def __init__(self, form, default_values=None):
+        super().__init__(form)
+        if default_values:
+            self.user_description.data = default_values.get('user_description')
```

Comment:
Apparently these lines are untested. Is there any case where we actually pass in default values?

Reply:
ah no there isn't I added those in testing some stuff out, i'll remove them 👍